.github/workflows/sync_vitessio_TEST.yml

```
@@ -0,0 +1,45 @@
+name: Sync Vitessio TEST
+
+### This is just a test workflow.
+### We just want a temporary workflow that can be run on push to verify access etc.
+### Currently we are running into problems with auth and GITHUB_TOKEN.
+### Once we have auth sorted, and can run this, we will delete this file and submit the non-test workflow.
+
+on:
+  push:
+    branches:
+      - sync-vitessio-workflow
+  workflow_dispatch:
+
```

```suggestion
permissions:
  contents: write
  
```

I think the auth issue you're running into is the GITHUB_TOKEN is read-only be default